cpp/zeder_to_zotero_importer.cc

```diff
@@ -334,9 +335,9 @@ unsigned ImportZederEntries(const Zeder::EntryCollection &zeder_entries, Harvest
             LOG_WARNING("couldn't import Zeder entry " + std::to_string(zeder_id) + " (" + title + "): already exists with different zeder id " + existing_title_section->getString("zeder_id"));
             continue;
         } else if (existing_journal_section == nullptr and autodetect_new_datasets) {
-            const std::string prodf(zeder_entry.getAttribute("prodf", ""));
-            if (prodf != "zota" && prodf != "zotat") {
-                LOG_DEBUG("Skipping Zeder entry " + std::to_string(zeder_id) + " (" + title + "): prodf is neither \"zota\" nor \"zotat\"");
+            if (upload_operation == Config::UPLOAD_OPERATION_TO_STRING_MAP.at(Config::UploadOperation::NONE)) {
```

Comment:
Why not fold this condition into the one above and save one more level of indentation?